app/assets/stylesheets/components/ui/_cookie.scss

```diff
@@ -15,26 +15,14 @@
 }
 
 .cookie-policy-content {
-  padding: 0 $baseline-spacing 0 0;
-  margin: 0 5px 0 0;
-  font-size: 0.875em;
+  @include column(12);
+  margin-top: $baseline-unit*3;
+  margin-bottom: $baseline-unit*3;
 }
 
-.cookie-submit {
-  position: absolute;
-  right: $baseline-spacing;
-  top: 0;
-  border: 0;
-  padding: 0;
-  background: $primary-blue-dark;
-  width: 19px;
-  height: 19px;
-  border-radius: 50%;
+.cookie-policy--close-button {
+  float: right;
+  font-size: 0.875rem;
+  padding: $baseline-unit $baseline-unit;
+  margin: 0 0 $baseline-unit $baseline-unit*3;
```

Comment:
These styles are being over-written by the ones for `button.btn`. Although we don't normally like to write styles like this we have done here so it may be necessary to define these like `button.cookie-submit `.

Reply:
Also, if the button has `font-size: 0.875rem;` I think the text should have the same font-size.